sensors_description/urdfs/sensors/realsense_d455.gazebo.xacro

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+
+<!--
+License: Apache 2.0. See LICENSE file in root directory.
+Copyright(c) 2017 PAL Robotics, S.L. All Rights Reserved
+
+This is the Gazebo URDF model for the Intel RealSense D435 camera
+-->
```

Comment:
Let's remove these or at least, let's write that these plugins are based on work done by XXX for the XXX camera.